modules/about-must-gather.adoc

```diff
@@ -24,7 +25,7 @@ Alternatively, you can collect specific information by running the command with
 +
 For example:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal,subs="attributes+"
```

Comment:
Hi @rohennes . The build should fail because of not closing the source block heading?

Reply:
Nice to test the bad Asciidoc stuff! 

No, the Vale review won't fail the build if it finds errors. It will only post review comments, so purely informational.